codec/doc.go

```diff
@@ -61,11 +61,10 @@ Rich Feature Set includes:
   - Drop-in replacement for encoding/json. `json:` key in struct tag supported.
   - Provides a RPC Server and Client Codec for net/rpc communication protocol.
   - Handle unique idiosyncrasies of codecs e.g.
-    - For messagepack, configure how ambiguities in handling raw bytes are resolved
-    - For messagepack, provide rpc server/client codec to support
-      msgpack-rpc protocol defined at:
-      https://github.com/msgpack-rpc/msgpack-rpc/blob/master/spec.md
-
+  - For messagepack, configure how ambiguities in handling raw bytes are resolved
```

Comment:
All this noise is from `gofmt`, unfortunately.

Reply:
Hiding whitespace helps a bit :)